nginx/server/sites.sls

```diff
@@ -10,13 +10,15 @@
 
 {%- if site.ssl.engine is not defined %}
 
-{%- if site.ssl.key is defined %}
+{%- if site.ssl.key is defined or site.ssl.key_file is defined %}
 
 {{ site.host.name }}_public_cert:
   file.managed:
   - name: /etc/ssl/certs/{{ site.host.name }}.crt
   {%- if site.ssl.cert is defined %}
   - contents_pillar: nginx:server:site:{{ site_name }}:ssl:cert
+  {%- elif site.ssl.cert_file is defined %}
+  - source: {{ site.ssl.cert_file }}
```

Comment:
This isn't expected behavior from using cert_file and key_file options - these options should simply point nginx or any service to use that files and do nothing more about them.